lib/screen/account/recovery_phrase_page.dart

```diff
@@ -6,28 +6,59 @@
 //
 
 import 'dart:io';
+import 'dart:ui';
 
+import 'package:autonomy_flutter/common/injector.dart';
 import 'package:autonomy_flutter/screen/app_router.dart';
+import 'package:autonomy_flutter/service/account_service.dart';
+import 'package:autonomy_flutter/service/cloud_service.dart';
+import 'package:autonomy_flutter/service/local_auth_service.dart';
 import 'package:autonomy_flutter/util/style.dart';
 import 'package:autonomy_flutter/view/back_appbar.dart';
+import 'package:autonomy_flutter/view/primary_button.dart';
 import 'package:autonomy_flutter/view/responsive.dart';
 import 'package:autonomy_theme/autonomy_theme.dart';
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:open_settings/open_settings.dart';
+import 'package:permission_handler/permission_handler.dart';
 
-class RecoveryPhrasePage extends StatelessWidget {
+class RecoveryPhrasePage extends StatefulWidget {
   final List<String> words;
 
   const RecoveryPhrasePage({Key? key, required this.words}) : super(key: key);
 
+  @override
+  State<RecoveryPhrasePage> createState() => _RecoveryPhrasePageState();
+}
+
+class _RecoveryPhrasePageState extends State<RecoveryPhrasePage> {
+  bool _isShow = false;
+  bool? _isBackUpAvailable;
+
+  @override
+  void initState() {
+    super.initState();
+    _checkBackUpAvailable();
```

Comment:
Use `WidgetsBinding.instance.addPostFrameCallback` for asynchronously loading data.